src/otx/algo/classification/efficientnet.py

```diff
@@ -131,7 +131,7 @@ def _exporter(self) -> OTXModelExporter:
             resize_mode="standard",
             pad_value=0,
             swap_rgb=False,
-            via_onnx=True,  # NOTE: This should be done via onnx
+            via_onnx=False,
```

Comment:
Could you give more explanations why direct export from openvino has been fine with non-tracing exception?

Reply:
I haven't faced any issues with these particular models, and I don't know why this flag was enabled here. If you have any problems with OVC on basic models, it should be reported and fixed in the next OV release.